pyEpiabm/pyEpiabm/core/person.py

```diff
@@ -167,3 +165,11 @@ def remove_place(self, place):
             ind = place_list.index(place)
             self.places.pop(ind)
             self.place_types.remove(place.place_type)
+
+    def close_place(self, closure_place_type):
+        if (hasattr(self.microcell, 'closure_start_time') is True) and (
```

Comment:
```suggestion
        if (hasattr(self.microcell, 'closure_start_time')) and (
```
hasattr should be boolean already, no need to compare it to true

Reply:
Fixed it in all files!